features/output_form.feature

```diff
@@ -63,30 +63,30 @@ Feature: Output Form
       | locality      | London         |
       | county        | Greater London |
       | post_code     | SW1H 9JJ       |
-    And I choose to add more claimants
-    And I fill in the fourth group claimant details with:
-      | field         | value          |
-      | title         | Mrs            |
-      | first_name    | Group4First    |
-      | last_name     | Group4Last     |
-      | date_of_birth | 21/11/1992     |
-      | building      | 110            |
-      | street        | Buckingham Pl  |
-      | locality      | London         |
-      | county        | Greater London |
-      | post_code     | SW1H 9JT       |
-    And I choose to add more claimants
-    And I fill in the fifth group claimant details with:
-      | field         | value          |
-      | title         | Mrs            |
-      | first_name    | Group5First    |
-      | last_name     | Group5Last     |
-      | date_of_birth | 21/10/1991     |
-      | building      | 112            |
-      | street        | Oxford Road    |
-      | locality      | London         |
-      | county        | Greater London |
-      | post_code     | SW1H 9JY       |
+#    And I choose to add more claimants
```

Comment:
Can we remove the commented out lines?

Reply:
Yes, well spotted - not sure if they need to be removed or re added - Ill check